Message in reaction roles, Stats fixing

vinc3nz opened the pull request:
Not yet done. Will doing this in the next days maybe.


**This code will mostly break the whole bot. This time it is really necessary to review the code**

vinc3nz commented:
Dan being funny again.

itzJOHv commented on bot/discord/events/reactionAdd.js:
```diff
@@ -35,7 +35,12 @@ module.exports = async(client, r, member) => {
         if (member.roles.cache.get(found.role) == null) {
             await member.roles.add(role.id);
-            member.user.send("Gave you the role: `" + role.name + "`!");
+            client.channels.cache.get("898041838701781013").send(`<@${member.id}>\nGave you the *${role.name}* role! To remove it click the reaction again.`)
+            .then(msg => {
+               msg.delete(10000)
+            })
+            .catch(e => console.log(e));
+            //member.user.send("Gave you the role: `" + role.name + "`!");
```
why did you need to make it like this?

itzJOHv commented on bot/discord/events/reactionRemove.js:
```diff
@@ -37,7 +37,12 @@ module.exports = async(client, r, member) => {
     if (member.roles.cache.get(found.role) != null) {
         await member.roles.remove(role);
-        member.user.send("Removed the role: `" + role.name + "`!");
+        client.channels.cache.get("898041838701781013").send(`<@${member.id}>\nRemoved the *${role.name}* role from you! To add it again click the reaction again.`)
+        .then(msg => {
+           msg.delete(10000)
+        })
+        .catch(e => console.log(e));
+        //member.user.send("Removed the role: `" + role.name + "`!");
```
same to this

omxpro commented:
Can you add confirm button to delete cmd

Sliden101 commented:
![image](https://user-images.githubusercontent.com/57210425/167647788-64112b11-d75b-4330-a4f6-aed0ef472d52.png)
deprecated

vinc3nz commented:
![image](https://media.discordapp.net/attachments/959714468135305227/973813915173675028/Screenshot_20220511_070504_com.discord.jpg)

Just adding a few more infos and the stats work again

vinc3nz commented:
@danielpmc
I think everything is done. But one question: How big are the disks? Are it about 300GB/server or are it Mb? Because I now multiply it by 1000 to show the size in Gb

danielpmc commented:
Its different per node. 

> @danielpmc I think everything is done. But one question: How big are the disks? Are it about 300GB/server or are it Mb? Because I now multiply it by 1000 to show the size in Gb

vinc3nz commented:
But for example Node 6 has 300GB and not 300MB?

vinc3nz commented:
I have removed the buttons I added.
The stats command should work fine. I will edit it over the time to provide single node infos etc.
In reaction roles there is a medsage semd saying you got the role. The message sill be deleted after a few seconds

DEV-DIBSTER commented:
Will be closing due to being outdated as well as conflicts.
